lib/client.js

```diff
@@ -158,6 +158,123 @@ class Client {
     const relativePathWithQuery = `${params.resourceType}?${new URLSearchParams(params.searchParams)}`;
     return this.httpClient.get(relativePathWithQuery);
   }
+
+  /**
+   * Create a resource by FHIR id.
```

Comment:
Is this right? We don't specify the ID as we create, right?

Reply:
Should be "Create a resource " or "Create a resource with a server assigned FHIR id"

Reply:
I'd definitely go with the former over the latter.